scripts/battlefields/Apollyon/nw_apollyon.lua

```diff
@@ -0,0 +1,571 @@
+        end,
+    },
+
+    {
+        mobs = { "Bardha" },
+        allDeath = function(mob)
+            depowerBoss(ID.NW_APOLLYON.mob.PLUTO)
+        end,
+
+        randomDeath = function(battlefield, mob)
+            xi.limbus.openDoor(battlefield, ID.npc.APOLLYON_NW_PORTAL[1])
+        end,
+    },
+
+    -- Floor 2
+    {
+        mobs = { "Zlatorog" },
+        mods =
+        {
+            [xi.mod.ATTP] = 200,
+            [xi.mod.ACC] = 100,
+        },
+
+        death = function(battlefield, mob, count)
+            xi.limbus.spawnFrom(mob, ID.NW_APOLLYON.npc.ITEM_CRATES[2])
+        end,
+    },
+
+    {
+        mobs = { "Mountain_Buffalo" },
+        allDeath = function(battlefield, mob)
+            depowerBoss(ID.NW_APOLLYON.mob.ZLATOROG)
+        end,
+
+        randomDeath = function(battlefield, mob)
+            xi.limbus.openDoor(battlefield, ID.npc.APOLLYON_NW_PORTAL[2])
+        end,
+    },
+
+    -- Floor 3
+    {
+        mobs = { "Millenary_Mossback" },
+        stationary = true,
+        mods =
+        {
+            [xi.mod.ATTP] = 200,
+            [xi.mod.ACC] = 100,
+        },
+
+        death = function(battlefield, mob, count)
+            xi.limbus.spawnFrom(mob, ID.NW_APOLLYON.npc.ITEM_CRATES[3])
+        end,
+    },
+
+    {
+        mobs = { "Apollyon_Scavenger" },
+        stationary = true,
+        allDeath = function(mob)
+            depowerBoss(ID.NW_APOLLYON.mob.MILLENARY_MOSSBACK)
+        end,
+
+        randomDeath = function(battlefield, mob)
+            xi.limbus.openDoor(battlefield, ID.npc.APOLLYON_NW_PORTAL[3])
+        end,
+    },
+
+    -- Floor 4
+    {
+        mobs = { "Cynoprosopi" },
+        mods =
+        {
+            [xi.mod.ATTP] = 200,
+            [xi.mod.ACC] = 100,
+        },
+
+        death = function(battlefield, mob, count)
+            xi.limbus.spawnFrom(mob, ID.NW_APOLLYON.npc.ITEM_CRATES[4])
+        end,
+    },
+    {
+        --
+        mobs = { "Gorynich" },
+        allDeath = function(battlefield, mob)
+            depowerBoss(ID.NW_APOLLYON.mob.CYNOPROSOPI)
+        end,
+
+        randomDeath = function(battlefield, mob)
+            xi.limbus.openDoor(battlefield, ID.npc.APOLLYON_NW_PORTAL[4])
+        end,
+    },
+
+    -- Floor 5
+    {
+        mobs = { "Kaiser_Behemoth" },
+        mobMods =
+        {
+            [xi.mobMod.ALLI_HATE] = 50,
+            [xi.mobMod.MAGIC_COOL] = 60,
+        },
+
+        mods =
+        {
+            [xi.mod.ATTP] = 200,
+            [xi.mod.ACC] = 100,
+        },
+
+        death = function(battlefield, mob, count)
+            npcUtil.showCrate(GetNPCByID(ID.NW_APOLLYON.npc.LOOT_CRATE))
+        end,
+    },
+
+    {
+        mobs = { "Kronprinz_Behemoth" },
+        mobMods = { [xi.mobMod.ALLI_HATE] = 50 },
+        stationary = false,
+        allDeath = function(battlefield, mob)
+            depowerBoss(ID.NW_APOLLYON.mob.KAISER_BEHEMOTH)
```

Comment:
Is there a source for this? bg wiki and ffxiclopedia don't mention this and I don't remember it, though its been years and I never did empirical testing. Is this intended to floor the bosses accuracy when that happens? that should be testable somewhat easily

Reply:
iirc, it links all remaining Kronprinz_Behemoth if you agro the KAISER_BEHEMOTH before killing them

Reply:
I'm asking my capture buddy to check on these on retail.

Reply:
ah, I was looking at that mobmods line with the alliance hate and thought it was setting alliance hate to facilitate the mass linkage, completely missed what depowerBoss() even did:
```lua
local depowerBoss = function(bossID)
    local boss = GetMobByID(bossID)
    boss:setMod(xi.mod.ATTP, 100)
    boss:setMod(xi.mod.ACC, 0)
end
```

Reply:
Looked back over the wiki and found this.
`The level bosses are weakened by killing all of the lesser mobs.`

My friend tested linking and they only link normally not super linking.

Reply:
Here's the capture: <https://www.youtube.com/watch?v=GgLP14QM7wQ>